.pylintrc

```diff
@@ -10,15 +10,15 @@ fail-under=10
 
 # Add files or directories to the blacklist. They should be base names, not
 # paths.
-ignore=CVS
+ignore=CVS,settings.py
```

Comment:
Why adding `settings.py` to the pylint ignore ? I would keep it as well in the linter, if there is a good reason to not lint it, then I would suggest to comment it in the pylintrc

Reply:
`settings.py` is not checked in anymore, it's a local user file and therefore we don't care about linting

Reply:
:+1: